### DISCUSSION TITLE
How to replicate the 'tokyodark.nvim' header image in README?

### DISCUSSION THREAD

**Discussion by shreyas-a-s** [tiagovla/tokyodark.nvim#37] [Q&A] [answered]
Hiya, Tiago.

The header image looks dope. How did you create it?

I mean this one:

![Screenshot_20240316-150527~2.png](https://github.com/tiagovla/tokyodark.nvim/assets/137637016/cfd6fbe6-2a61-4305-b9b0-12b4ff532dfd)

**Comment by tiagovla** [comment, marked as answer]
I did it in Inkscape. It's just a gradient with the colorscheme colors. I had the original svg somewhere.

**Reply by shreyas-a-s** [reply]
Cool. Now I gotta brush up my inkscape skills.

**Comment by tiagovla** [comment]
The svg one, just in case.
![tokyodark](https://github.com/tiagovla/tokyodark.nvim/assets/30515389/780d9837-7ea7-46c9-85f6-1ed8f2eabb8d)

**Reply by shreyas-a-s** [reply]
Thanks, man. Taking time to find the image and sent it, I appreciate that.
